index.html

```diff
@@ -295,8 +304,7 @@
 	</div>
 
 	<!-- buttons auf map -->
-	<button id="pauseButton" onClick="rettich_motion_keys('pause')">Pause</button>
-	<button id="resumeButton" onClick="rettich_motion_keys('resume')">Resume</button>
+	<button id="toggleButton" onClick="toggleFunction('pause')">&#9658; / &#9208; </button>
```

Comment:
Wie funkt das? Was sind das für hasches?

Reply:
Das sind die zeichen für play und stopp

Reply:
Also einfach unicode. 
Als das noch von pause zu resume gewechselt ist hat sich immer die größe des buttons geändert

Reply:
Hier wird 'pause' übergeben, aber die funktion hat keine Parameter?

Reply:
Ah ja stimmt das kann raus

Reply:
Habs geändert

Reply:
Ich fänd ja cool, wenn das ein Button wäre und sich das Icon je nach Aktion anpasst. Also wenn man pause machen kann dann sind || und wenn man play machen kann ist da |>

Reply:
Aber das ist dann bisschen unelegant weil ich dann in alle funktionen die das starten auch das reinschreiben muss. Also das geht sicherlich eleganter aber das ist dann aufwändiger

Reply:
Habs eigentlich eingearbeitet aber habe git probleme :D